src/components/DisplayUTXOs.jsx

```diff
@@ -71,28 +80,30 @@ const Utxo = ({ utxo, ...props }) => {
               <div>
                 <small className="text-secondary">{utxo.confirmations} Confirmations</small>
               </div>
-              <div>
-                <rb.Button
-                  size="sm"
-                  variant={utxo.frozen ? 'outline-warning' : 'outline-info'}
-                  disabled={isSending}
-                  onClick={() => onClickFreeze(utxo)}
-                >
-                  {isSending && (
-                    <rb.Spinner
-                      as="span"
-                      animation="border"
-                      size="sm"
-                      role="status"
-                      aria-hidden="true"
-                      className="ms-1 me-2"
-                    />
-                  )}
-                  {utxo.frozen
-                    ? t('current_wallet_advanced.button_unfreeze')
-                    : t('current_wallet_advanced.button_freeze')}
-                </rb.Button>
-              </div>
+              {showFreezeActionButton && (
+                <div>
+                  <rb.Button
+                    size="sm"
+                    variant={utxo.frozen ? 'outline-warning' : 'outline-info'}
+                    disabled={isSending || !isOperationEnabled()}
```

Comment:
The disabled state for this button doesn't really look like a disabled button imo:

### Enabled:
<img width="176" alt="Screenshot 2022-04-15 at 16 17 16" src="https://user-images.githubusercontent.com/10026790/163581819-33f49ad3-8a35-4680-ae15-3cb905c17562.png">

### Disabled:
<img width="158" alt="Screenshot 2022-04-15 at 16 17 24" src="https://user-images.githubusercontent.com/10026790/163581842-16073fd3-5a41-4c1d-8f3e-c1001f4dc33a.png">

Maybe we just hide the button? Since there's no explanation on why freezing is disabled, we could just remove the buttons altogether as long as they cannot be clicked. What d you think?

Reply:
Yes, very valid. Will do!

Reply:
Done!